CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- [Inline evaluation results not visible in Light themes](https://github.com/BetterThanTomorrow/calva/issues/1855)
```

Comment:
Should we add "Fix:" before this?

Reply:
Yes. And I saw that you already did it. Thanks!